lib/connectors_shared/extraction_utils.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
```

Comment:
I was surprised to see this here, but I'm seeing it's used in the base Confluence adapter. We'll have to decide how to extract it from the crawler. I think the crawler was intentionally designed to depend on a very limited set of gems, and doesn't currently rely on _connectors_sdk_.

I expect this file has no meaningful changes from its origin in Enterprise Search. Let me know if that's not the case.

Don't forget the copyright text.

Reply:
This also surprised me. When I dug into it, I found we've been using the utility like this since 2017. I also found this comment:

https://github.com/elastic/ent-search/blob/045d996b7468c0734e6ce0fa3ef2d74be40e1189/lib/extractor_utils.rb#L1-L5

which made me think we should just have this copied here, AND in crawler, and not have either depend on the other.

Reply:
Noted. I will keep one copy in `connectors` and leave it unchanged in `ent-search`.